File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    count1 = 0
    count2 = 0
    for i in num_list:
        if i % 2 == 0:
            count1 += 1
        else:
            count2 += 1
    count = [count1,count2]
    return count

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] solution(int[] num_list) {
        int evenCount = 0;
        int oddCount = 0;

        // 주어진 배열을 순회하면서 짝수와 홀수의 개수를 센다.
        for (int num : num_list) {
            if (num % 2 == 0) {
                evenCount++;
            } else {
                oddCount++;
            }
        }

        // 결과 배열을 생성하고 짝수와 홀수의 개수를 담는다.
        int[] result = {evenCount, oddCount};

        return result;
    }
}
